examples/users/update_user.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+me = EasyPost::User.retrieve_me
+
+me.recharge_threshold = '50.00'
+
+me.save
+
+puts me
```

Comment:
Let's not use `me` as variable name here, you could rename it to something like `user`